taxonomies/food/categories.txt

```diff
@@ -111884,3 +111884,6 @@ tr:Hazır gıda çeşitlilik paketi
 gpc_category_code:en:10000613
 gpc_category_name:en:Prepared/Preserved Foods Variety Packs
 gpc_category_description:en:Definition: Includes any products that can be described/observed as two or more distinct Prepared and Preserved Foods sold together which exist within the schema but belong to different classes, that is two or more products contained within the same pack which cross classes within the Prepared and Preserved Foods Family. Includes products such as Chicken Korma and Ice Cream variety packs. Items that are received free with purchases should be removed from the classification decision-making process. Definition Excludes: Excludes products such as Jam and Chocolate Spread variety packs and Chips and Dips variety packs.
+
+mozzarella:Cheeses:Cheeses by type:Cheeses by milk:Cheeses from animal milks:Cheeses from cow milk:Fresh cheeses:Mozzarella
+minimum_number_of_ingredients:en: 3
```

Comment:
Mozzarella entry already exist ! You just have to add your property. This line is not compatible with taxonomy (see https://wiki.openfoodfacts.org/Global_taxonomies)